display.go

```diff
@@ -0,0 +1,21 @@
+package drivers
+
+import "image/color"
+
+type Display interface {
+	// Size returns the current size of the display.
+	Size() (x, y int16)
+
+	// SetPizel modifies the internal buffer.
+	SetPixel(x, y int16, c color.RGBA)
+
+	// Display sends the buffer (if any) to the screen.
+	Display() error
+
+	// DisplayPixel sends a single pixel to the screen.
+	DisplayPixel(x, y int16, c color.RGBA)
+
+	// DisplayRect sends this part of the buffer to the screen for incremental updates
+	// (e.g. button press animations, blinking cursor, etc.).
+	DisplayRect(x1, y1, x2, y2 int16)
```

Comment:
Was @aykevl suggesting to remove this?

Reply:
Here https://github.com/tinygo-org/drivers/pull/31#issuecomment-473262223

In some devices it might not be possible to update just a part of the display, I'm thinking about the SSD1306 or the PCD8544, both use a byte to represent a 8 pixel column and it's the minimum you can send to the display, you will end up updating a bigger area (you need to calculate that new area). In other devices, such a function might not exists. I think it might be useful, it's easy to fallback and update the whole screen (or a bigger area) but the results might not be what you expected.